### ISSUE TITLE
Content of the Transferee object

### ISSUE THREAD

**Issue by pimg** [dcsaorg/DCSA-EBL-Envelope#8]
We should allow for sending additional information about a transferee so additional verification can be performed. These include things like LEI, Tax reference numbers, VAT numbers, etc. 

The DCSA Party object could be a good fit and could be evaluated for use:  [DCSA Party object](https://app.swaggerhub.com/domains/dcsaorg/DOCUMENTATION_DOMAIN/2.0.1#/components/schemas/party)

**Comment by pimg** [comment]
This issue is related to: #12

**Comment by enriquesempe** [comment]
> We should allow for sending additional information about a transferee so additional verification can be performed. These include things like LEI, Tax reference numbers, VAT numbers, etc.
> 
> The DCSA Party object could be a good fit and could be evaluated for use: [DCSA Party object](https://app.swaggerhub.com/domains/dcsaorg/DOCUMENTATION_DOMAIN/2.0.1#/components/schemas/party)

I don't think it would be necessary to send additional information, because by using the GET /v1/parties API (please refer to issue #13), the sender can validate the information themselves. In addition, by sending something like the LEI, there might be the chance that the receiver platform does not have that information for that specific party so nothing can be validated at all.

**Comment by pimg** [comment]
at a very minimal this is indeed optional. But agreed that if the information is indeed already coming from the receiving platform simply sending this back to them does not make much sense.
